documentation/src/main/asciidoc/topics/code_examples/HotRodConnectionPools.java

```diff
@@ -2,8 +2,12 @@
 clientBuilder
    .addServer()
     .host("127.0.0.1")
-    .port(11222);
-   .connectionPool() <1>
-    .maxActive(10) <2>
-    .minIdle(20); <3>
+    .port(11222)
+   //Configure client connection pools.
+   .connectionPool()
```

Comment:
These lines aren't aligned. Space is missing.
```suggestion
    //Configure client connection pools.
    .connectionPool()
```